Revit_Core_Engine/Query/Passes.cs

```diff
@@ -253,9 +253,10 @@ public static bool Passes(this Parameter parameter, FilterByParameterNumber requ
             if (!CheckIfNotNull(parameter, request))
                 return false;
 
+
+            double paramValue;
```

Comment:
You can get rid of the `else` statement easily
```suggestion
            double paramValue = double.NaN;
```